src/storage/txn/store.rs

```diff
@@ -11,13 +11,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use super::{Error, Result};
 use kvproto::kvrpcpb::IsolationLevel;
+
 use storage::mvcc::{
     BackwardScanner, BackwardScannerBuilder, ForwardScanner, ForwardScannerBuilder,
 };
 use storage::mvcc::{Error as MvccError, MvccReader};
-use storage::{Key, KvPair, ScanMode, Snapshot, Statistics, Value};
+use storage::{Key, KvPair, Snapshot, Statistics, Value};
+
+use super::{Error, Result};
+
+pub trait Store {
```

Comment:
Maybe we can rename it to `StoreReader`? There are too much `Store`.

Reply:
Good idea!

Reply:
Seems that too many places uses `Store`, for example, `SnapshotStore` should be renamed to `SnapshotStoreReader` as well if we would like to change this name. Let's defer it to another PR?